### PR TITLE
Jakartize platform/ApplicationClients.adoc

### PR DESCRIPTION
**Pull request by xstefank**
Signed-off-by: xstefank <xstefank122@gmail.com>

One question - how should references to JAR be handled. Is it also trademarked? I repeated the approach from [ApplicationAssemblyDeployment.adoc](https://github.com/eclipse-ee4j/jakartaee-platform/blob/master/specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc) but I am unsure about "Jakarta Archive (JAR)" usage.

### CODE REVIEW

File: specification/src/main/asciidoc/platform/ApplicationClients.adoc [hunk 1]
```diff
@@ -2,10 +2,10 @@
 == Application Clients
 
 This chapter describes application clients
-in the Java™ Platform, Enterprise Edition (Java EE).
+in the Jakarta™ Platform, Enterprise Edition (Jakarta EE).
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationClients.adoc]: I would just use "Jakarta(TM) Enterprise Edition" (drop the Platform).

File: specification/src/main/asciidoc/platform/ApplicationClients.adoc [hunk 2]
```diff
@@ -101,13 +102,13 @@ application client main class. Because the application client container
...
 into the application client class uses _static_ fields and methods,
-unlike other Java EE components. Injection occurs before the _main_
+unlike other Jakarta EE components. Injection occurs before the _main_
 method is called.
 
 === Application Programming Interfaces
 
 Application clients have all the facilities of
-the JavaTM Platform, Standard Edition (subject to security
+the Java™ Platform, Standard Edition (subject to security
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationClients.adoc]: nice catch, thanks

File: specification/src/main/asciidoc/platform/ApplicationClients.adoc [hunk 3]
```diff
@@ -121,15 +122,15 @@ initialize the name service client library, and so on).
 [[a3315]]
 === Packaging and Deployment
 
-Application clients are packaged in JAR format
+Application clients are packaged in Jakarta Archive (JAR) format
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationClients.adoc]: Leave JAR as-is

File: specification/src/main/asciidoc/platform/ApplicationClients.adoc [hunk 4]
```diff
@@ -232,10 +233,10 @@ descriptor for the module.
 If the value of the _metadata-complete_
 attribute is specified as _“true”_ , the deployment tool must ignore any
 annotations that specify such deployment information in the class files
-packaged in the application client jar file. Such annotations must also
+packaged in the application client JAR file. Such annotations must also
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/ApplicationClients.adoc]: leave lowercase "jar"

**Comment by xstefank** [comment]
@kwsutter updated, thanks!

**Pull request merged**